### DISCUSSION TITLE
remove live stream footer

### DISCUSSION THREAD

**Discussion by sheepdogs** [owncast/owncast#2873] [General Owncast discussion]
Is there a way to remove the "Stream is online. 08:22:44 1 viewer" from the footer of the live feed?

Where is the code for it located?

**Comment by gabek** [comment]
With CSS you're able to make any kind of user interface modifications you'd like, including removing entire components you don't want. https://owncast.online/docs/website/#custom-styles-via-css

**Comment by sheepdogs** [comment]
Thanks ... this did it!

#stream-viewer-count {
  display: none;
}

**Reply by gabek** [reply]
In v0.1.0 there's actually a toggle for displaying viewer count since it's common for people to not want to display it.
